examples/vnet/main.c

```diff
@@ -81,18 +81,130 @@ static void vnf_adjust_mbuf(struct rte_mbuf *m, struct doca_pkt_info *pinfo)
     //rte_pktmbuf_adj(m,diff);
 }
 
+static uint64_t doca_timer_period = 1;
```

Comment:
we have gw_port , lets put it there and leave main small file

Reply:
I means print stats function

Reply:
done